examples/mysensor/README.md

```diff
@@ -2,7 +2,52 @@
 
 This example demonstrates a user defining a new sensor model.
 
+
+## How to add custom models
+
+Models can be added to the viam-server by creating a struct that implements the interface of the subtype we want to register.
```

Comment:
probably not the place for this, but our codebase seems to be inconsistent about model vs subtype (error messages often mention subtypes :((()

Reply:
ya, it could probably be better